bigtable/use-cases/fraudDetection/src/test/java/FraudDetectionTestUtil.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class FraudDetectionTestUtil {
+
+  // Make sure that the variable is set from running Terraform.
+  public static void requireVar(String varName) {
+    assertNotNull(varName);
+  }
+
+  // Make sure that the required environment variables are set before running the tests.
+  public static String requireEnv(String varName) {
+    String value = System.getenv(varName);
+    assertNotNull(
+        String.format("Environment variable '%s' is required to perform these tests.", varName),
+        value);
+    return value;
+  }
+
+  // Parse Terraform output and populate the variables needed for testing.
+  private static void parseTerraformOutput(Process terraformProcess) throws IOException {
+    BufferedReader reader =
+        new BufferedReader(new InputStreamReader(terraformProcess.getInputStream()));
+
+    // Process terraform output.
+    String line;
+    while ((line = reader.readLine()) != null) {
+      System.out.println(line);
+      if (line.contains("pubsub_input_topic = ")) {
+        // example: pubsub_output_topic = "THE NAME WE WANT"
```

Comment:
I assume the linter asked for this? If not, we can remove, otherwise keep. However, if we are keeping, I would suggest using example names here that resemble the actual format of the variables instead of just NAME WE WANT. It can be helpful to understand what to expect. 

I'm not sure how it is formatted for pubsub but at least for bigtable table it could be "my-table-name" or for the GCS bucket "gs://my-bucket" (this is somewhere it is useful to know if the gs:// prefix is included or excluded for example)